src/test/java/io/strimzi/kafka/bridge/mqtt/clients/KafkaConsumerClient.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.clients;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Properties;
+
+/**
+ * A Kafka consumer client
+ */
+public class KafkaConsumerClient {
```

Comment:
Why we need this class it's not doing that much, just wrapping and calling the underneath Kafka consumer client. Any specific reasons for it?

Reply:
Because of the consumer properties? I want to leave the MqttBridgeIT cleaner. So we wrap everything needed for the consumer in a class.
To be honest I wanted to do the same for the Mqtt client, its configuration is simpler.